domene/src/main/java/no/nav/foreldrepenger/info/tjenester/DokumentforsendelseTjenesteImpl.java

```diff
@@ -259,6 +256,7 @@ private ForsendelseStatusDataDTO getForsendelseStatusDataDTO(Behandling behandli
     }
 
     private SøknadXmlDto mapTilSøknadXml(List<MottattDokument> dokumenter) {
+        if (dokumenter.isEmpty()) return null;
```

Comment:
Bedre å returnere Optional.Empty i stedet for null

Reply:
Tenker du jeg da bør gå for optionals også i de to metodene som mapTilSøknadXml bruker?

Reply:
Jeg tenker at man ikke burde returnere null fra en metode, uavhengig av hvor den kalles fra

Reply:
Droppet det, fant en pen nok variant.